crates/biome_css_analyze/tests/specs/nursery/noInvalidPositionAtImportRule/validComment.css

```diff
@@ -0,0 +1,2 @@
+/* some comment */
+@import 'foo.css';
```

Comment:
nit

Maybe add a test case for

```css
@charset "utf-8";
/* some comment */
@import 'foo.css';
```

Reply:
fix: https://github.com/biomejs/biome/pull/2717/commits/aa879c161bece5a8123250083cb4591b2e7accc7